src/main/scala/lasp/hapi/service/latis2/Latis2Interpreter.scala

```diff
@@ -61,4 +67,32 @@ class Latis2Interpreter[F[_]: Sync](dir: String) extends HapiInterpreter[F] {
       // Get the metadata.
       metadata <- EitherT.liftF(Latis2Util.getDatasetMetadata(dataset))
     } yield metadata
+
+  override def getData(req: DataRequest): F[dm.Dataset] = req match {
+    case DataRequest(id, minTime, maxTime, params, _, _) =>
+      val ops = List(
```

Comment:
You can add a `format_time` operation to the latis request to ensure that you get the times back in the required format.

Reply:
Adding this to the list of operations appears to work:

```scala
TimeFormatter("yyyy-MM-dd'T'hh:mm:ss.SSS'Z'")
```

Reply:
If we're always converting time to that format, we can also always report that the length is 25 and that the units are UTC.

The coverage data is trickier. We'd need to read those values and potentially convert them into the appropriate format.